### PR TITLE
Freshen up dev readme

### PR DESCRIPTION
**Pull request by Isaac-DeFrain**
Small improvements to the dev readme

### CODE REVIEW

**Comment by Isaac-DeFrain** [comment]
CI won't work because this branch lives outside of the MinaProtocol/mina repo. I do not have write access.

File: README-dev.md [hunk 1]
```diff
@@ -26,19 +26,19 @@ Quick start instructions:
...
 
-3.  Pull in the submodules: 
+3.  Pull in the submodules:
 
     ```sh
     git submodule update --init --recursive
     ```
 
-    If this command fails with `git@github.com: Permission denied (publickey).` then you need to set up SSH keys on your machine. Follow the [Generating a new SSH key and adding it to the ssh-agent](https://help.github.com/en/articles/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent) instructions.
+If this command fails with `git@github.com:Permission denied (publickey).` then you need to set up SSH keys on your machine. Follow the [Generating a new SSH key and adding it to the ssh-agent](https://help.github.com/en/articles/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent) instructions.
```
Comment (mrmr1993) [inline README-dev.md:41]: I don't think you want to remove the indentation here.

**Review by mrmr1993 — COMMENTED** [review]
See comment; lgtm otherwise

File: README-dev.md (hunk 1 above)
Comment (Isaac-DeFrain) [reply README-dev.md:41]: Good catch!

**Comment by rbonichon** [comment]
Closing as superseded by #15273

**Comment by mrmr1993** [comment]
Thank you @Isaac-DeFrain, this has now finally landed in #15273.

**Pull request closed without merging**